FPWD/2023-05-23/index.html

```diff
@@ -252,6 +252,33 @@
         "name": "W3C"
       }
     },
+    {
+      "id": "https://w3c-ccg.github.io/data-integrity-spec/",
+      "type": "TechArticle",
+      "name": "Data Integrity. Manu Sporny; Dave Longley. Credentials Community Group. CG-DRAFT",
+      "url": "https://w3c-ccg.github.io/data-integrity-spec/",
+      "publisher": {
+        "name": "W3C"
+      }
+    },
+    {
+      "id": "https://www.w3.org/TR/vc-jwt/",
```

Comment:
same comment here... https://github.com/w3c/vc-di-bbs/blob/main/index.html#L100

Reply:
don't see this one?